cluster-up/cluster/ocp-4.3/provider.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e
+
+image="openshift-cnv/kubevirtci-ocp-4.3@sha256:446265401bff3e90548f34461f0814493f52489c82e4527844250f7a02dcbb7c"
+
+source ${KUBEVIRTCI_PATH}/cluster/ephemeral-provider-common.sh
+
+function _port() {
+    ${_cli} ports --prefix $provider_prefix --container-name cluster "$@"
+}
+
+function _install_from_cluster() {
+    local src_cid="$1"
+    local src_file="$2"
+    local dst_perms="$3"
+    local dst_file="${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/$4"
+
+    touch $dst_file
+    chmod $dst_perms $dst_file
+    docker exec $src_cid cat $src_file > $dst_file
+}
+
+function up() {
+    workers=$(($KUBEVIRT_NUM_NODES-1))
+    if [[ ( $workers < 1 ) ]]; then
+        workers=1
+    fi
+    echo "Number of workers: $workers"
+    params="--random-ports --background --prefix $provider_prefix --master-cpu 6 --workers-cpu 6 --secondary-nics ${KUBEVIRT_NUM_SECONDARY_NICS} --registry-volume $(_registry_volume) --workers $workers --container-registry quay.io ${image}"
+    if [[ ! -z "${RHEL_NFS_DIR}" ]]; then
+        params=" --nfs-data $RHEL_NFS_DIR ${params}"
+    fi
+
+    if [[ ! -z "${OKD_CONSOLE_PORT}" ]]; then
+        params=" --ocp-console-port $OKD_CONSOLE_PORT ${params}"
+    fi
+
+    ${_cli} run okd ${params}
```

Comment:
Maybe we should add `ocp` at a later time instead of reusing `okd` here?

Reply:
Well is lready using all the OKD scripts, let's just add it in case it gets very different